src/Refitter.Tests/Examples/TrimUnusedSchemaTests.cs

```diff
@@ -0,0 +1,214 @@
+﻿using FluentAssertions;
+
+using Refitter.Core;
+using Refitter.Tests.Build;
+
+using Xunit;
+
+namespace Refitter.Tests.Examples;
+
+public class TrimUnusedSchemaTests
+{
+    private const string OpenApiSpec = @"
+openapi: 3.0.1
+paths:
+  /v1/Warehouses:
+    post:
+      tags:
+        - Warehouses
+      operationId: CreateWarehouse
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Warehouse'
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Warehouse'
+        '400':
```

Comment:
I'm guilty of defining a `ProblemDetails` component but never specifying it as a `400` response content 😅